psga/sars_cov_2/psga.nf

```diff
@@ -103,7 +98,7 @@ workflow psga {
             // files are FASTA
             ch_fasta_files = stage_sample_file(ch_metadata_records_single_fasta)
             // mock ncov
-            ch_ncov_qc_csv = Channel.fromPath('/mock_file')
```

Comment:
awsbatch executor does not like this. Easy fix anyway. The new code is also more correct.